css/index.css

```diff
@@ -143,4 +270,47 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+.about-page {
+	line-height: 1.25;
+}
+
+section .top {
+	border-bottom: 1px dashed black;
+	padding-bottom: 20px;
+	margin-bottom: 20px;
+}
+
+div .top {
+	margin-top: 20px;
+}
+.middle-content .about-page {
+	display: flex;
+	flex-flow: row wrap;
+
+}
+.middle-content img {
+	display: flex;
+}
+div .middle-content .about-page {
+	border-bottom: 1px dashed black;
+	display: flex;
```

Comment:
`display: flex` and `flex-flow: column wrap` are not needed here. `display:flex` is only needed on the parent element, and the column wrap doesn't seem to make a change in chrome dev tools.